packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -88,24 +87,52 @@ const HelperText = styled(Text).attrs({
 
 function VideoOptionsPanel({ selectedElements, pushUpdate }) {
   const { isTranscodingEnabled } = useFFmpeg();
-  const { muteExistingVideo } = useLocalMedia((state) => ({
-    muteExistingVideo: state.actions.muteExistingVideo,
-  }));
+  const {
+    muteExistingVideo,
+    isResourceTrimming,
+    isResourceMuting,
+    isResourceProcessing,
+    isResourceProcessingById,
+  } = useLocalMedia(
+    ({
+      state: {
+        isResourceProcessing,
+        isResourceProcessingById,
+        isResourceMuting,
+        isResourceTrimming,
+      },
+      actions: { muteExistingVideo },
+    }) => ({
+      isResourceProcessing,
+      isResourceProcessingById,
+      isResourceMuting,
+      isResourceTrimming,
+      muteExistingVideo,
+    })
+  );
   const resource = getCommonValue(selectedElements, 'resource');
-  const { isMuting, isMuted, isTrimming } = resource;
+  const { isMuted, id: resourceId = 0 } = resource;
+  const isTrimming = isResourceTrimming(resourceId);
+  const isMuting = isResourceMuting(resourceId);
   const loop = getCommonValue(selectedElements, 'loop');
   const isSingleElement = selectedElements.length === 1;
 
   const handleMuteVideo = useCallback(() => {
     muteExistingVideo({ resource });
   }, [resource, muteExistingVideo]);
 
-  const shouldDisableVideoActions = !canTranscodeResource(resource);
+  const shouldDisableVideoActions =
+    isResourceProcessing(resourceId) ||
+    isResourceProcessingById(resourceId) ||
```

Comment:
Isn't there a way to just check `isResourceProcessing` once? It seems a bit add that it needs to be checked twice by the same resource ID. What's the difference there?

Reply:
Has to do with naming, see https://github.com/google/web-stories-wp/pull/9785#discussion_r758331003 and other thread at https://github.com/google/web-stories-wp/pull/9785#discussion_r758309753

Reply:
One is for if the current resource is being processed and the other new resource that is linked original resource.